website/docs/api-reference/hooks/use-query-loader.md

```diff
@@ -90,7 +90,7 @@ A tuple containing the following values:
     * Parameters
         * `variables`: the variables with which the query is loaded.
         * `options`: `LoadQueryOptions`. An optional options object, containing the following keys:
-            * `fetchPolicy`: _*[Optional]*_ Determines if cached data should be used, and when to send a network request based on the cached data that is currently available in the Relay store (for more details, see our [Fetch Policies](https://www.internalfb.com/intern/wiki/Relay/guided-tour-of-relay/reusing-cached-data-for-rendering/#fetch-policies) and [Garbage Collection](https://www.internalfb.com/intern/wiki/Relay/guided-tour-of-relay/reusing-cached-data-for-rendering/#garbage-collection-in-re) guides):
+            * `fetchPolicy`: _*[Optional]*_ Determines if cached data should be used, and when to send a network request based on the cached data that is currently available in the Relay store (for more details, see our [Fetch Policies](../../guided-tour/reusing-cached-data/fetch-policies) and [Garbage Collection](../../guided-tour/reusing-cached-data/presence-of-data) guides):
```

Comment:
Same links as https://github.com/facebook/relay/blob/b17907beb3b46b05a4639af1d2af2382aa56ab0e/website/docs/api-reference/hooks/use-lazy-load-query.md?plain=1#L47 
I also confirmed those links by running websites locally and browsing http://localhost:3000/docs/next/api-reference/use-query-loader/ .